lib/uplinkc/uplink_definitions.h

```diff
@@ -4,14 +4,65 @@
 #include <stdbool.h>
 #include <stdlib.h>
 
-typedef struct APIKey   { long _handle; } APIKey;
-typedef struct Uplink   { long _handle; } Uplink;
-typedef struct Project  { long _handle; } Project;
+typedef struct APIKey   { long _handle; } APIKeyRef_t;
+typedef struct Uplink   { long _handle; } UplinkRef_t;
+typedef struct Project  { long _handle; } ProjectRef_t;
+typedef struct Bucket   { long _handle; } BucketRef_t;
```

Comment:
Note from posix, `_t` are reserved. http://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html

Reply:
But also everybody seems to be ignoring that. :/

Reply:
https://stackoverflow.com/a/3225396/192220